src/network/networkd-dhcp4.c

```diff
@@ -52,6 +52,39 @@ void network_adjust_dhcp4(Network *network) {
                 network->dhcp_client_identifier = network->dhcp_anonymize ? DHCP_CLIENT_ID_MAC : DHCP_CLIENT_ID_DUID;
 }
 
+static int dhcp4_get_static_routes(Link *link, sd_dhcp_route ***ret_routes, size_t *ret_num) {
```

Comment:
This returns a list of routes from either sd_dhcp_lease_get_classless_routes or  sd_dhcp_lease_get_static_routes. So we call the first set first "classless routes" and then "static classless routes", and then merge that with the second set "static routes" into "static routes". This is very confusing. Please keep the naming consistent with the `sd_dhcp_lease_get_*` functions.

Reply:
More precisely, classless static routes + static routes into static routes. haha.
`dhcp4_get_classless_static_or_static_routes()` ?? That sounds slightly long and redundant for me, but ok.